{"uid":"2ab1391121527a1","name":"test_create_user_with_valid_values","fullName":"Test_case.test_create_user#test_create_user_with_valid_values","historyId":"c4528f1fcc5809fc1e39d09a583baa8d","time":{"start":1703345474516,"stop":1703345475155,"duration":639},"status":"failed","statusMessage":"AssertionError: User already exist\nassert 422 == 201","statusTrace":"set_environment_variable = 'stag'\n\n    @allure.severity(severity_level=\"Critical\")\n    def test_create_user_with_valid_values(set_environment_variable):\n        endpoint = TestData().get_endpoint_env(\"stag\", 'create_user')\n        payload = TestData().payload_method()\n        header = TestData().header_method()\n    \n        payload['id'] = '5818268'\n        payload['name'] = 'ra__erma'\n        payload['email'] = 'ddrrtyrdd@gmail.com'\n        payload['gender'] = 'male'\n        payload['status'] = 'active'\n        payload_json = json.dumps(payload)\n        response = requests.request(\"POST\", endpoint, headers=header, data=payload_json)\n        res_dict = response.json()\n        response_code = response.status_code\n>       assert response_code == 201, \"User already exist\"\nE       AssertionError: User already exist\nE       assert 422 == 201\n\nTest_case\\test_create_user.py:36: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"set_environment_variable","time":{"start":1703345474515,"stop":1703345474515,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"afterStages":[{"name":"set_environment_variable::0","time":{"start":1703345475155,"stop":1703345475155,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentStep":false,"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"parentSuite","value":"Test_case"},{"name":"suite","value":"test_create_user"},{"name":"host","value":"devshakya"},{"name":"thread","value":"2660-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Test_case.test_create_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":5},"items":[{"uid":"3f9e019b3a5b7dbb","reportUrl":"http://localhost:8080/job/cyanConnode_api_project/18/allure/#testresult/3f9e019b3a5b7dbb","status":"failed","statusDetails":"AssertionError: User already exist\nassert 422 == 201","time":{"start":1703345138859,"stop":1703345139405,"duration":546}},{"uid":"9d1731802fc07084","reportUrl":"http://localhost:8080/job/cyanConnode_api_project/16/allure/#testresult/9d1731802fc07084","status":"failed","statusDetails":"AssertionError: User already exist\nassert 422 == 201","time":{"start":1703345070031,"stop":1703345070753,"duration":722}},{"uid":"547706e7e871f8c7","reportUrl":"http://localhost:8080/job/cyanConnode_api_project/13/allure/#testresult/547706e7e871f8c7","status":"failed","statusDetails":"AssertionError: User already exist\nassert 422 == 201","time":{"start":1703344697885,"stop":1703344698393,"duration":508}},{"uid":"8cff8e1991a6cefe","reportUrl":"http://localhost:8080/job/cyanConnode_api_project/12/allure/#testresult/8cff8e1991a6cefe","status":"passed","time":{"start":1703344563612,"stop":1703344564229,"duration":617}}]},"tags":[]},"source":"2ab1391121527a1.json","parameterValues":[]}